src/App.js

```diff
@@ -1,25 +1,83 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+import "bootstrap/dist/css/bootstrap.css";
+import "./App.css";
+import ContactList from "./ContactList";
+import { Route, Switch } from "react-router-dom";
+import ContactProfile from "./ContactProfile";
+import AddContact from "./AddContact";
+import EditContact from "./EditContact";
+
+export const ContactContext = React.createContext();
 
 function App() {
+  const [contacts, setContacts] = useState(contactsApi.contacts);
+
+  const contactContextValue = {
```

Comment:
I like that you used context for practice-sake, but in this case, you're only passing this state one level deeper than this component, so it would have been just as easy (probably) to use regular ol' state.